README.md

```diff
@@ -36,36 +47,33 @@ particular emphasis is placed on code quality and reproducibility.
 We aim to support open source compilers and libraries released within two years of each QMCPACK release. Use of software versions
 over two years old may work but is discouraged and untested. Proprietary compilers (Intel, NVHPC) are generally supported over the
 same period but may require use of an exact version. We also aim to support the standard software environments on machines such as
-Summit at OLCF, Theta at ALCF, and Cori at NERSC. Use of the most recently released compilers and library versions is particularly
-encouraged for highest performance and easiest configuration.
+Frontier and Summit at OLCF, Aurora and Polaris at ALCF, and Perlmutter at NERSC. Use of the most recently released compilers and
+library versions is particularly encouraged for highest performance and easiest configuration.
 
-Nightly testing currently includes the following software versions on x86:
+Nightly testing currently includes at least the following software versions:
 
 * Compilers
-  * GCC 11.2.0, 9.2.0
-  * Clang/LLVM 13.0.0
-  * Intel 19.1.1.217 configured to use C++ library from GCC 9.1.0 
-  * NVIDIA HPC SDK 21.5 configured to use C++ library from GCC 9.1.0
-* Boost 1.77.0, 1.68.0
-* HDF5 1.12.1
+  * GCC 13.2.0, 11.4.0
+  * Clang/LLVM 16.0.6
+* Boost 1.83.0, 1.77.0
+* HDF5 1.14.2
 * FFTW 3.3.10, 3.3.8
-* CMake 3.21.1, 3.15.0
+* CMake 3.27.4, 3.21.3
 * MPI
-  * OpenMPI 4.1.1, 3.1.6
-  * Intel MPI 19.1.1.217
-* CUDA 11.4
+  * OpenMPI 4.1.5
+* CUDA 11.2
 
-Workflow tests are performed with Quantum Espresso v6.8.0 and PySCF v1.7.5. These check trial wavefunction generation and
-conversion through to actual QMC runs.
+GitHub Actions-based tests include additional version combinations from within our two year support window. On a developmental basis
+we also check the latest Clang and GCC development versions, AMD Clang and Intel OneAPI compilers. 
 
-On a developmental basis we also check the latest Clang and GCC development versions, AMD AOMP and Intel OneAPI compilers.
+Workflow tests are currently performed with Quantum Espresso v7.2.0 and PySCF v2.2.0. These check trial wavefunction generation and
+conversion through to actual QMC runs.
 
 # Building with CMake
 
- The build system for QMCPACK is based on CMake.  It will auto-configure based on the detected compilers and libraries. Previously
- QMCPACK made extensive use of toolchains, but the system has since been updated to eliminate the use of toolchain files for most
- cases.  Specific compile options can be specified either through specific environment or CMake variables.  When the libraries are
- installed in standard locations, e.g., /usr, /usr/local, there is no need to set environment or CMake variables for the packages.
+ The build system for QMCPACK is based on CMake.  It will auto-configure based on the detected compilers and libraries. Specific
+ compile options can be specified either through specific environment or CMake variables.  When the libraries are installed in
```

Comment:
The sentence 'Specific compile options ...' is a little unclear to me.  In the previous version, it's contrasting with toolchains, but now that the sentence about toolchains was removed, I'm not sure what it is trying to convey.